lib/PAUSE/package.pm

```diff
@@ -83,12 +83,27 @@ sub give_regdowner_perms {
   my $self = shift;
   my $dbh = $self->connect;
   my $package = $self->{PACKAGE};
+  my $main_package = $self->{MAIN_PACKAGE};
   local($dbh->{RaiseError}) = 0;
   # warn "Going to execute [SELECT userid FROM mods WHERE modid = '$package']";
-  my ($sth_mods) = $dbh->selectrow_hashref("SELECT userid
+  my $sql = "SELECT userid
                                 FROM   mods
-                                WHERE  modid = ?", undef, $package);
-  if ($sth_mods) { # make sure we regard the owner as the owner
+                                WHERE  modid = ?
+                                ";
+  my @args = ($package);
+  unless(lc($main_package) eq lc($package)) {
+    $sql .= q{
+    UNION
+    SELECT userid
+    FROM   perms
+    WHERE  LOWER(package) = LOWER(?)
```

Comment:
I tried to be consistent with the existing code with regards to the case sensitivity.  Plus don't the tests use SQLite so it needs to work in a DB server independent manner?

No lc on the modid was a case of not touching existing code any more than I needed to.  I didn't even consider touching that query.

I rather suspect that that whole bit will be chopped out as I think it's related to that 03modules thing that's effectively been removed.  Touching it further is probably a waste of time.